### PR TITLE
dev-embedded/ponyprog: bump version to 3.1.3, qhexedit2 to 0.8.9

### PR DESCRIPTION
**Pull request by ktrace**
- bump version to 3.1.3, qhexedit to 0.8.9
- switch to vanile Simsys/qhexedit

Package-Manager: Portage-3.0.30, Repoman-3.0.
Signed-off-by: Victor Kustov <ktrace@yandex.ru>

### CODE REVIEW

**Comment by ktrace** [comment]
So, https://github.com/gentoo/gentoo/pull/25042 seems not needed

File: app-editors/qhexedit2/qhexedit2-0.8.9_p20210525.ebuild [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~riscv ~x86"
+IUSE="doc +gui python"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+#PATCHES=(
+#	"${FILESDIR}/${PN}-0.8.4-setup.py.patch"
+#)
```
Comment (juippis) [inline app-editors/qhexedit2/qhexedit2-0.8.9_p20210525.ebuild:25]: Can be removed (I can do it too)

File: app-editors/qhexedit2/qhexedit2-0.8.9_p20210525.ebuild [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+RDEPEND="
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtwidgets:5
+	python? (
+		dev-python/PyQt5[gui,widgets,${PYTHON_USEDEP}]
+		<dev-python/sip-5[${PYTHON_USEDEP}]
```
Comment (juippis) [inline app-editors/qhexedit2/qhexedit2-0.8.9_p20210525.ebuild:33]: ... but please verify this first. Is this **absolutely** needed?

Comment (ktrace) [reply app-editors/qhexedit2/qhexedit2-0.8.9_p20210525.ebuild]: sip5 - no. remaked.

**Review by juippis — APPROVED** [review]
```
 * Messages for package dev-embedded/ponyprog-3.1.3:
 * QA Notice: package is installing udev rules without inheriting udev.eclass!
 * Packages must inherit udev.eclass then call udev_reload in pkg_postinst.
>>> Auto-cleaning packages...
```
I'll fix, thanks!

**Pull request closed without merging**